pkg/apis/core/validation/validation.go

```diff
@@ -6871,6 +6875,23 @@ func ValidateLoadBalancerStatus(status *core.LoadBalancerStatus, fldPath *field.
 				allErrs = append(allErrs, field.Invalid(idxPath.Child("ip"), ingress.IP, "must be a valid IP address"))
 			}
 		}
+
+		if utilfeature.DefaultFeatureGate.Enabled(features.LoadBalancerIPMode) {
```

Comment:
We don't usually check gates like this in validation - we check them in `dropDisabled`, and then here we do:

```
if gate enabled && value == nil {
    error - required
} else if value != nil && invalid value {
    error - notSupported
}
```

Then, wehn the gate goes away it becomes:

```
if value == nil {
    error - required
} else if invalid value {
    error - notSupported
}
```